tests/e2e/test_exit.py

```
@@ -0,0 +1,130 @@
+import os
```

Ruff

`os` imported but unused

Remove unused import: `os`

(F401)